libdevcrypto/SecretStore.cpp

```diff
@@ -246,6 +260,24 @@ h128 SecretStore::readKeyContent(string const& _content, string const& _file)
 	}
 }
 
+bool SecretStore::recode(Address const& _address, string const& _newPass, function<string()> const& _pass, KDF _kdf)
+{
+	bytesSec s = secret(_address, _pass);
+	if (s.empty())
+		return false;
+
+	for (auto& k: m_keys)
```

Comment:
Does it make sense to put that into a wrapper that returns a pointer to an element of m_keys given an address?